cloud/scope/vcn_reconciler.go

```diff
@@ -49,7 +49,7 @@ func (s *ClusterScope) ReconcileVCN(ctx context.Context) error {
 }
 
 func (s *ClusterScope) IsVcnEquals(actual *core.Vcn, desired infrastructurev1beta1.VCN) bool {
-	if *actual.DisplayName == desired.Name && s.IsTagsEqual(actual.FreeformTags, actual.DefinedTags) {
+	if *actual.DisplayName == desired.Name {
 		return true
 	}
 	return false
```

Comment:
Nothing to fix, but I did notice some of the `IsXEquals` methods are `!=` with the last return is `true` and this one is `==` with a last return of `false`. As we keep moving forward we should keep the logic roughly consistent.

Reply:
fixed